test/extended/templates/helpers.go

```diff
@@ -79,109 +73,64 @@ func setUser(cli *exutil.CLI, user *userapi.User) {
 	}
 }
 
-// EnsureTSB makes sure the TSB is present where expected and returns a client to speak to it and
-// and a close method which provides the proxy.  The caller must call the close method, usually done in AfterEach
+// EnsureTSB makes sure a TSB is present where expected and returns a client to
+// speak to it and a close method which provides the proxy.  The caller must
+// call the close method, usually done in AfterEach
 func EnsureTSB(tsbOC *exutil.CLI) (osbclient.Client, func() error) {
-	exists := true
-	if _, err := tsbOC.AdminKubeClient().Extensions().DaemonSets(tsbNS).Get("apiserver", metav1.GetOptions{}); err != nil {
-		if !kerrors.IsNotFound(err) {
-			o.Expect(err).NotTo(o.HaveOccurred())
-		}
-		exists = false
-	}
+	configPath := exutil.FixturePath("..", "..", "examples", "templateservicebroker", "templateservicebroker-template.yaml")
+
+	err := tsbOC.AsAdmin().Run("new-app").Args(configPath, "-p", "LOGLEVEL=4", "-p", "NAMESPACE="+tsbOC.Namespace()).Execute()
```

Comment:
I was surprised to find this change rebasing another pull.  `oc cluster up` and the ansible installer will use the predictable behavior of `oc process` and either `oc create` or `oc apply`.  Neither will use `oc new-app`.  This seems like the kind of drift that @bparees was concerned about before.  Why make this change?

Reply:
There were several areas in this routine which I believe were problematic, including the one which I believe which was responsible for the test failures we were seeing last week.  In the process of aiming to fix all the areas I identified, I also made the decision to use oc new-app here to reduce lines of code and to match what is done in many other tests owned by devex.

Reply:
@deads2k and I discussed this at lunch.  I don't know if he's convinced but i'm ok with this.  I'm worried about drift in the files being used to deploy the TSB (the ansible installer has an extra file that's not used here so things can get out of sync), i'm not so concerned that we're using two different "template instantiation" mechanisms since if we have critical differences in how templates get instantiated w/ one mechanism vs another, we have bigger problems.